### ISSUE TITLE
New point release

### ISSUE THREAD

**Issue by ockham** [jadejs/jade#2170]
Sorry for nagging ;-) Following up from #2169, I'd appreciate if you could release a point release, so I can update my project's dependency on Jade to a version that includes that fix.

**Comment by TimothyGu** [comment]
The current Jade release does not have an dependency on jstransformer-uglify-js. It's dependency on uglify-js is `^2.4.19` includes 2.6.1.

**Comment by ockham** [comment]
Yeah, but it depends on [`"transformers": "2.1.0"`](https://github.com/jadejs/jade/blob/1.11.0/package.json#L32), which in turn depends on [`"uglify-js": "~2.2.5"`](https://github.com/ForbesLindesay/transformers/blob/2.1.0/package.json#L66) -- so I'm stuck with uglify-js 2.2.x through that dependency chain :-(

**Comment by TimothyGu** [comment]
@ockham, it's not in our place to be able to update to a version of transformers that uses an up-to-date UglifyJS because of backwards compatibility. File a ticket at transformers requesting a 2.1.1 that uses an up-to-date UglifyJS, but really there's no point in doing so since we are releasing 2.0.0 Soon™.

**Comment by Y-LyN-10** [comment]
transformers has become deprecated. What about a replacement with jstransformer? It doesn't depend on uglify-js at all :)
